escapades_backend/config/application.rb

```diff
@@ -45,5 +45,10 @@ class Application < Rails::Application
     config.session_store :cookie_store, key: '_interslice_session'
     config.middleware.use ActionDispatch::Cookies
     config.middleware.use config.session_store, config.session_options
+
+    if ['development', 'test'].include? ENV['RAILS_ENV']
+      Dotenv::Railtie.load
+    end
+    HOSTNAME = ENV['HOSTNAME']
```

Comment:
So we use ENV or secrets?

Reply:
both, I tried to use secrets for cloudinary but it didnt work. I believe cloudinary works only with.env (no mention of credentials setup in their documentation)

Reply:
but youre right this part is useless